project_task_scheduling/views/project_task_scheduling_views.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).-->
+<odoo>
+
+    <record id="project_task_scheduling_view_form" model="ir.ui.view">
+        <field name="name">project.task.scheduling.form</field>
+        <field name="model">project.task.scheduling</field>
+        <field name="arch" type="xml">
+            <form>
+                <sheet>
+                    <group>
+                        <field name="proposal_id"/>
+                        <field name="employee_id"/>
+                        <field name="task_id"/>
+                    </group>
+                    <group>
+                        <field name="datetime_start"/>
+                        <field name="datetime_stop"/>
+                    </group>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+    <record id="project_task_scheduling_view_tree" model="ir.ui.view">
+        <field name="name">project.task.scheduling.tree</field>
+        <field name="model">project.task.scheduling</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="proposal_id"/>
+                <field name="employee_id"/>
+                <field name="task_id"/>
+                <field name="datetime_start"/>
+                <field name="datetime_stop"/>
+                <field name="delayed"/>
+            </tree>
+        </field>
+    </record>
+
+    <record id="project_task_scheduling_view_timeline" model="ir.ui.view">
+        <field name="name">project.task.scheduling.timeline</field>
+        <field name="model">project.task.scheduling</field>
+        <field name="arch" type="xml">
+            <timeline date_start="datetime_start"
```

Comment:
Would be nice to also print the picture of the employee here.

Reply:
Yes, we could use the same timeline definition as the one in project_timeline + rest of the extras. The dependency handling is another thing in the roadmap. I'm not sure if directly depend on all the other modules or develop several modules that plugs on the main one.

Reply:
Well, when you have 50 or 100 task a picture is not needed. Because you need to see global over view of the planned schedule, so as much compacted is everything, better visualization.

Reply:
Well, when you have 50 or 100 task a picture is not needed. Because you need to see global over view of the planned schedule, so as much compacted is everything, better visualization.